src/scripts/main.js

```diff
@@ -1,7 +1,41 @@
 'use strict';
 
 const pushNotification = (posTop, posRight, title, description, type) => {
-  // write code here
+  const div = document.createElement('div');
+
+  div.setAttribute('class', 'notification');
+  div.style.top = `${posTop}px`;
+  div.style.right = `${posRight}px`;
+
+  const newTitle = document.createElement('h2');
+
+  newTitle.setAttribute('class', 'title');
+  newTitle.textContent = title;
+
+  const newDescription = document.createElement('p');
+
+  newDescription.textContent = description;
+
+  div.appendChild(newTitle);
+  div.appendChild(newDescription);
+
+  if (type === 'success') {
+    div.classList.add('success');
+  }
+
+  if (type === 'error') {
+    div.classList.add('error');
+  }
+
+  if (type === 'warning') {
+    div.classList.add('warning');
+  }
+
+  document.body.appendChild(div);
+
+  setTimeout(() => {
+    div.style.display = 'none';
+  }, 2000);
```

Comment:
The `setTimeout` function hides the notification after 2000 milliseconds (2 seconds). Consider making this duration configurable by adding an additional parameter to the `pushNotification` function. This would allow for more flexible notification durations.